packages/http/libraries/core/src/http/adapter/InversifyHttpAdapter.ts

```diff
@@ -32,10 +32,20 @@ export abstract class InversifyHttpAdapter<
   TResult,
 > {
   protected readonly httpAdapterOptions: InternalHttpAdapterOptions;
+  readonly #awaitableRequestMethodParamTypes: Set<RequestMethodParameterType>;
   readonly #container: Container;
   readonly #logger: Logger;
 
-  constructor(container: Container, httpAdapterOptions?: HttpAdapterOptions) {
+  constructor(
+    container: Container,
+    httpAdapterOptions: HttpAdapterOptions | undefined,
+    awaitableRequestMethodParamTypes:
```

Comment:
Maybe, could this parameter be optional? So we could avoid undefined at some child contructors(majority in this case)